hwbench/bench/monitoring.py

```diff
@@ -187,22 +205,31 @@ def next_iter():
 
             start_bmc = self.get_monotonic_clock()
```

Comment:
You kept the `start_bmc` name, is it expected?

Reply:
Yes because this is just before __monitor_bmc(). We measure here the time spent at monitoring the BMC.

Reply:
Ah ok I get it, I had missed that `end_monitoring` is overwritten if we have pdu monitoring just after calling `__monitor_pdus`, so it has two uses.